www/md_base/src/icons/gear-outline.svg

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Ok, I think I understand more. Its okay if the svg are concatenated.

What should be avoided is to include icons directly in git, we should rather use bower to fetch those icons in bower package.

This will need update of guanlecoja to support configuration of which icons we want in the bower section:

"material-design-icons":
      version: "1.0.1"
      files: [ "notification/svg/production/ic_system_update_48px.svg", "notification/svg/production/ic_more_48px.svg" ]